README.md

```diff
@@ -4,6 +4,34 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio?style=social)](https://twitter.com/strimziio)
 
+# Deprecation notice
```

Comment:
How can people use the Kafka provider in clients? Is it part of the Kafka clients JAR / dependency? Or do they need to add something more? I think this would be worth mentioning here as there are users using it that way.

Reply:
Good point, it's indeed something that is worth mentioning.
Kafka's config provider is in the kafka-clients JAR, so as long as users are using at least 3.5.0 in their applications, they should be able to switch over by just replacing the class name.

I pushed an update that adds some details.